examples/js/cli.js

```diff
@@ -47,7 +47,7 @@ const ccxt         = require ('../../ccxt.js')
     , { execSync } = require ('child_process')
```

Comment:
@pcriadoperez, with the current design, we can't do this transparently for the user, right? (reconnect without him noticing it)

Reply:
@carlosmiei Unfortunately not. We have to resolve or reject the promises for the code on user land to continue. And resolving an empty value results in a typerror most of the times so thought it was better to reject with the specific error explaining the reset.

The only way I though we could make it unnoticeable would be to close the connection, open it, and then redo all of the users subscriptions. But in that case we would have to change all the current subscriptions to correctly store them to be able to redo them. Also we risk losing any updates in between, and generally seemed like a lot of change for just this feature, so thought the current implementation would be best, so the user is also aware.